packages/shaku-code-annotate/src/__tests__/index.test.ts

```diff
@@ -141,7 +141,86 @@ describe("parseLine() can parse comment lines", () => {
     const func = () => parseLine("   @highlight unexpected ");
     expect(func).toThrow();
   });
-
+  test("   @dim start", async () => {
+    const result = parseLine("   @dim start ");
+    expect(result).toEqual({
+      type: "DirectiveDim",
+      config: {
+        mark: "start",
+      },
+    });
+  });
+  test("   @dim end", async () => {
+    const result = parseLine("   @dim end ");
+    expect(result).toEqual({
+      type: "DirectiveDim",
+      config: {
+        mark: "end",
+      },
+    });
+  });
+  test("   @dim ", async () => {
+    const result = parseLine("   @dim ");
+    expect(result).toEqual({
+      type: "DirectiveDim",
+      config: {
+        mark: "below",
+      },
+    });
+  });
+  test("   @dim below ", async () => {
+    const result = parseLine("   @dim below ");
+    expect(result).toEqual({
+      type: "DirectiveDim",
+      config: {
+        mark: "below",
+      },
+    });
+  });
+  test("   @dim unexpected", async () => {
+    const func = () => parseLine("   @dim unexpected ");
+    expect(func).toThrow();
+  });
+  test("   @focus start", async () => {
```

Comment:
You'd better keep PR small as possible, let's put @focus into another PR

Reply:
> You'd better keep PR small as possible, let's put `@focus` into another PR


Okay, I will open another PR.